lib/jxl/decode.cc

```diff
@@ -1887,12 +1887,11 @@ JXL_EXPORT JxlDecoderStatus JxlDecoderImageOutBufferSize(
   JxlDecoderStatus status = PrepareSizeCheck(dec, format, &bits);
   if (status != JXL_DEC_SUCCESS) return status;
 
-  size_t row_size =
-      jxl::DivCeil(dec->metadata.size.xsize() * format->num_channels * bits,
-                   jxl::kBitsPerByte);
```

Comment:
there's similar code in JxlDecoderPreviewOutBufferSize and JxlDecoderDCOutBufferSize, it should probably be changed the same way, if this change is needed (see other comment)